build.gradle

```diff
@@ -111,6 +111,7 @@ task install(type: Upload) {
 test {
     dependsOn install
     systemProperty "local.repo", localRepo.toURI()
+    maxParallelForks = 2
```

Comment:
Did this actually improve anything for you?  I think I tried this out at one point and decided that with it running the android builds under the covers, it didn't actually speed things up (i.e. the bottleneck is IO and more parallelization in the tests didn't do anything to work around that).

Reply:
Yeah, you are right. There's almost no difference here in terms of performance of the tests. 
max parallel forks enabled:
https://scans.gradle.com/s/zmoo7agh4z23a/performance/build
max parallel forks disabled:
https://scans.gradle.com/s/ikgh3usjl3wwc/performance/build

Will remove.